website/pages/404.jsx

```diff
@@ -15,18 +18,39 @@ export default function NotFound() {
       })
   }, [])
 
-  return (
-    <div id="p-404" className="g-grid-container">
-      <h1 className="g-type-display-1">Page Not Found</h1>
+  const defaultMessage = (
+    <>
       <p>
-        We&lsquo;re sorry but we can&lsquo;t find the page you&lsquo;re looking
+        We&rsquo;re sorry but we can&rsquo;t find the page you&rsquo;re looking
         for.
       </p>
       <p>
         <Link href="/">
           <a>Back to Home</a>
         </Link>
       </p>
+    </>
+  )
+
+  const reg = /\/v\d+[.]\d+[.](\d+|x)/g
```

Comment:
Can we use the util from `docs-page`? https://github.com/hashicorp/react-components/blob/main/packages/docs-page/util/index.ts

Reply:
`router.query` unfortunately doesn't play nicely with this util, since it's empty (`{}`) on page refresh,

a possible alternative is splitting `asPath` into "params", but that feels over-complicated